### PR TITLE
java: Cleanup generator output

### PR DESCRIPTION
**Pull request by brettkail-wk**
### Story:
Commits are intended to be reviewed independently.  See commit messages for details.  Most of the commits are best reviewed with whitespace ignored.

### Acceptance Criteria:
- [ ] At least one InfRe Squad 2 member has reviewed and +1'd
- [ ] Code has been tested and results documented
- [ ] Unit tests have been updated
- [ ] Updates to documentation if necessary
- [ ] Verify and document changes to any other Messaging components
- [ ] Pull request made against the 'develop' branch, not master

### My Test Results:
Manually tested against a repo with 3k lines of Frugal, and pulled it into a repo that exhaustively tests all the various constructs.  The changes reduce the size of the JAR of the API repo by ~25%.

#### Reviewers:
@Workiva/product2

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: test/expected/java/actual_base/nested_thing.java [hunk 1]
```diff
@@ -286,11 +286,7 @@ public String toString() {
 		boolean first = true;
 
 		sb.append("things:");
-		if (this.things == null) {
```
Comment (sagesmith-wf) [inline test/expected/java/actual_base/nested_thing.java]: Nice

**Comment by charliestrawn-wf** [comment]
lgtm 👀

**Comment by brianshannan-wf** [comment]
@Workiva/release-management-p

**Pull request merged**